### ISSUE TITLE
Check colour contrast

### ISSUE THREAD

**Issue by notionparallax** [bvn-architecture/transformative#12]
http://snook.ca/technical/colour_contrast/colour.html#fg=B3B3B3,bg=1D1D1D

This is the regular text over the regular background. can we make them more contrasty?

The text in the editorial on the main page will definitely need to be more contrasty

**Comment by patriktyppo** [comment]
Try to set font-weight to 600 first (ONLY FOR BODY TXT). It looks really thin right now. This should increase readability. If that's not enough then we can add contrast (#6e6e6e)

**Comment by notionparallax** [comment]
## editorial

@600:
![image](https://cloud.githubusercontent.com/assets/1865196/6860059/84e06c48-d476-11e4-98d3-b2717611433e.png)
regular
![image](https://cloud.githubusercontent.com/assets/1865196/6860066/96f133b8-d476-11e4-8dbf-dbbf20c19042.png)

Either way that text will fail accessibility testing. It doesn't become compliant against that background until rgb(207, 207, 207)  or #CFCFCF.

p.s. I'm doing all the colours in rgb

**Comment by patriktyppo** [comment]
I think with 600 text is quite good! If we go #cfcfcf that means that body txt is brighter than headings/links and we do not want that. We want brighter headings and links and body txt that is comfortable to read on dark background (means not too bright or too dark).

sample #cfcfcf
![image](https://cloud.githubusercontent.com/assets/11640099/6860390/8c1de31e-d47b-11e4-8c76-703667bcf37e.png)
I think this is too bright and actually harder to read. We need to find the happy medium with these too...

**Comment by notionparallax** [comment]
Did you look at the accessibility standards checker?
We need to be able to say that the site is standards compliant.
The current text everywhere else only gets a value of "sort of" compliant.
If we can't make a website accessibility compliant what hope do we have of
convincing clients that we can do an accessible building!

**Comment by patriktyppo** [comment]
Accessibility standards checker by who? Who says what is the absolutely truth with txt contrast?

I quickly googled accessibility standards cause I do not know which one you are preferring and I got this result for the site:
![image](https://cloud.githubusercontent.com/assets/11640099/6861684/58bb064c-d48e-11e4-9674-fc2f128efcf1.png)

If you only accept designs through some/one checker you need to tell me. Otherwise I'm wasting my time doing any other kind of designs. 

I'm happy to change colors/contrast but it needs to be in balance and readability needs to be good. Too bright is not better readability. Should I do couple tests and find a good combination or are you absolute with your standard?

**Comment by notionparallax** [comment]
http://khan.github.io/tota11y/

This is useful. 

headings
rgba(230, 230, 230, 1);
body
rgba(190, 190, 190, 1);

looks pretty good and is plenty contrasty

**Comment by patriktyppo** [comment]
With these values there is not enough difference between headings and body. This makes it really hard  to read and spot things (all looks white). Also this contrast is starting to hurt your eyes. It's better to have grey with not pure white txt than black with pure white txt. Plenty of contrast is not better readability, when dealing with dark background. Also a lot of other things effects readability than just contrast. And the reason why we have dark background is that Transformative site will look different enough of our upcoming BVN website which is going to be white.

We could try values:
headings rgb 200, 200, 200
body rgb 130, 130, 130

This would give more contrast without going overboard.
